### ISSUE TITLE
Wildcard Branches

### ISSUE THREAD

**Issue by halkeye** [toolmantim/release-drafter#197]
on https://github.com/jenkinsci/blueocean-plugin we use master for the next release (its 1.x.0) and releases/vfoo for 1.foo.1 type patches) 

The branches mentioned in the readme, digging through the code it doesn't seem like they support wildcards. Do they? do you want a patch to do so?

**Comment by toolmantim** [comment]
Yep, they don’t support wildcards I’m afraid! But I’d be happy to accept support for it.
